tizen-api/src/tizen-api-pipeline.c

```diff
@@ -116,25 +120,52 @@ cb_sink_event (GstElement * e, GstBuffer * b, gpointer data)
           memcpy (&elem->tensorsinfo, &tconfig.info, sizeof (GstTensorsInfo));
           elem->size = 0;
 
-          g_assert (elem->tensorsinfo.num_tensors == num_mems);
+          if (elem->tensorsinfo.num_tensors != num_mems) {
+            dlog_print (DLOG_ERROR, "nnstreamer-capi-pipeline",
+                "The sink event of [%s] cannot be handled because the tensor type mismatches.",
+                elem->name);
+            gst_object_unref (caps);
```

Comment:
better to use gst_caps_unref()

Reply:
OOoooh. that's a bug fixed with #1266. Anyway, I'll fix it up with this commit.